Esri/arcgis-maps-sdk-swift-samples
# [New] Show service area

## Description

This PR implements **`Show service area`** in `Analysis` category.

`swift/issues/5546`

## How To Test

Open view and load map. Add in facilities and barriers to map and then tap show service. The service graphics should render correctly in relation facilities and barriers. 

## Screenshots


![show-service-area](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/13937686/3d30fbcf-e1b2-49ae-962b-8286ef05bf93)
